src/Symfony/Component/DependencyInjection/Attribute/AsAlias.php

```diff
@@ -19,6 +19,10 @@
 #[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
 final class AsAlias
 {
+    /**
+     * @param string|null   $id     The id of the service to alias
```

Comment:
You can move the comment before each parameter for the promoted properties. Like in Cache class. So that you have more space for description and example if necessary.

Reply:
Addressed this comment as well as the other ones :slightly_smiling_face: